pkg/domain/infra/abi/play.go

```diff
@@ -605,6 +609,15 @@ func (ic *ContainerEngine) playKubePod(ctx context.Context, podName string, podY
 		}
 	}
 
+	cfg, err := ic.Libpod.GetConfigNoCopy()
+	if err != nil {
+		return nil, nil, err
+	}
+	var readOnly types.OptionalBool
+	if cfg.Containers.ReadOnly {
+		readOnly = types.NewOptionalBool(cfg.Containers.ReadOnly)
```

Comment:
decl-setting it directly `readOnly := types.NewOptionalBool(cfg.Containers.ReadOnly)` without the if statement would probably be faster.

Reply:
Since false is the default, we want to leave as an optionalbool without being set, that way the settings from the kube yaml take over.  If I set this OptionalBool to False, then it ignores the kube.yaml settings.

Reply:
Ah, that makes sense, thanks!